src/bin/sol-fbp-generator/main.c

```diff
@@ -1611,12 +1611,12 @@ generate(struct sol_vector *fbp_data_vector)
 
     /* Header name is currently inferred from the module name. */
     SOL_VECTOR_FOREACH_IDX (&ctx->headers, header, i) {
-        out("#include \"%.*s\"\n", SOL_STR_SLICE_PRINT(*header));
+        out("#include <%.*s>\n", SOL_STR_SLICE_PRINT(*header));
```

Comment:
This one may be problematic, as some of the node types used could have been generated locally and not be installed.